### PR TITLE
Add tracing to wsoc servers to debug quiesce error

### PR DESCRIPTION
**Pull request by volosied**
Should have added these in PR #16645

### CODE REVIEW

**Comment by volosied** [comment]
#build

File: dev/io.openliberty.wsoc.internal_fat/publish/servers/traceTestServer/server.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!--
-    Copyright (c) 2014, 2016 IBM Corporation and others.
+    Copyright (c) 2014, 2021 IBM Corporation and others.
```
Comment (volosied) [inline dev/io.openliberty.wsoc.internal_fat/publish/servers/traceTestServer/server.xml:2]: forgot to add copyright in other PR

**Review by loriadi — APPROVED** [review]
Thanks, Volod.

**Comment by volosied** [comment]
#run-libby-bot

**Pull request merged**